.github/workflows/PagerCI.yml

```
@@ -0,0 +1,247 @@
+name: Pager CI
+
+on: [push]
+
+jobs:
+  install-dependencies:
+    name: Install dependencies
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: yarn
```

```suggestion
        run: yarn install
```
to keep consistency with other steps